README.md

```diff
@@ -65,12 +76,10 @@ Anyone and everyone is welcome to [contribute](.github/CONTRIBUTING.md). We do a
 Please see our [guidelines for contributing doc](https://github.com/Esri/contributing/blob/master/README.md)
 
 ## Licensing
-Copyright 2022 Esri
-
-Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at
-
-http://www.apache.org/licenses/LICENSE-2.0
 
+Copyright 2023 Esri
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+You may obtain a copy of the License <http://www.apache.org/licenses/LICENSE-2.0>
```

Comment:
Revert these changes as the copyright is not planned to be updated. I believe the last discussion was to use the same copyright year at the time of the creation of the sample.

Reply:
Interesting! But our 200.1.0 samples use the 2023 license right?

Reply:
Also, line 82 address code inspection warning on using http links in the .md files.

Reply:
> But our 200.1.0 samples use the 2023 license right?

Not quite, samples added to the repo in 2022 use `Copyright 2022` and samples added to the repo in 2023 use `Copyright 2023`. Or at least, they should be 😅 

> Also, line 82 addresses GitHub [README_style_checker](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/blob/main/tools/CI/README_Metadata_StyleCheck/README_style_checker.py), warning on using http links in the .md files.

As regards the licensing, there are ESRI guidelines on the format of licensing. You can visit any platform samples repo to notice a `LICENSE` file and the license text at the bottom. In fact, should see in all public ESRI repositories. The GitHub checker is helpful, but in order to change this text it probably will need approval first.

Reply:
The last line makes sense, good to know.